packages/ui-kit/tailwind.config.js

```diff
@@ -8,8 +8,8 @@ const spacing = getSpacing(
   0.4,
   "rem",
   [
-    0, 0.5, 1, 1.5, 2, 2.5, 3, 4, 5, 6, 6.5, 7, 8, 10, 11, 12, 14, 16, 18, 19, 21, 22, 28, 85, 256,
-    350,
+    0, 0.5, 1, 1.5, 2, 2.5, 3, 3.25, 4, 5, 6, 6.5, 7, 8, 10, 11, 12, 14, 16, 18, 19, 21, 22, 28, 85,
```

Comment:
:thonking_face: Isn't `3` good enough?

Reply:
Not if we want it to look like in the designs